alchemist/alchemist-incarnation-scafi/src/main/scala/it/unibo/alchemist/model/scafi/ScafiIncarnationForAlchemist.scala

```diff
@@ -0,0 +1,10 @@
+package it.unibo.alchemist.model.scafi
+
+import it.unibo.scafi.incarnations.BasicAbstractIncarnation
+
+/**
+  * @author Roberto Casadei
```

Comment:
This must be killed, as per Checkstyle rule (the DVCS commits are used to identify the author)